dwave_networkx/drawing/chimera_layout.py

```diff
@@ -230,87 +229,86 @@ def draw_chimera(G, **kwargs):
 
 
 def draw_chimera_embedding(G, *args, **kwargs):
-    """Draws an embedding onto the chimera graph G, according to layout.
-
-    If interaction_edges is not None, then only display the couplers in that
-    list.  If embedded_graph is not None, the only display the couplers between
-    chains with intended couplings according to embedded_graph.
+    """Draws an embedding onto the chimera graph ``G``.
 
     Parameters
     ----------
     G : NetworkX graph
-        Should be a Chimera graph or a subgraph of a Chimera graph.
+        :term:`Chimera` :term:`graph` or a :term:`subgraph` of a Chimera graph.
 
     emb : dict
-        A dict of chains associated with each node in G.  Should be
-        of the form {node: chain, ...}.  Chains should be iterables
-        of qubit labels (qubits are nodes in G).
+        Embedding ``emb`` for all nodes of ``G`` as a dict of chains
+        of the form ``{node: chain, ...}``. where chains are iterables
+        of qubit labels. Qubits are nodes in ``G``.
 
     embedded_graph : NetworkX graph (optional, default None)
-        A graph which contains all keys of emb as nodes.  If specified,
-        edges of G will be considered interactions if and only if they
-        exist between two chains of emb if their keys are connected by
-        an edge in embedded_graph
+        Graph which contains all keys of the ``emb`` parameter as nodes. If specified,
+        the edges of ``G`` will be considered interactions if and only if they
+        exist between two chains of the emb parameter and if their keys are connected by
+        an edge in the ``embedded_graph`` parameter; only the couplers between
+        chains with intended couplings are displayed.
```

Comment:
"chains with intended couplings are displayed." --> what does "intended couplings" mean?

Reply:
I am not sure; it is from line 235, which I tried to integrate into the parameter description:
> If embedded_graph is not None, the only display the couplers between
> chains with intended couplings according to embedded_graph.
Do you have a suggestion?

Reply:
I understand the meaning there to be that only the couplers for edges of ``G`` that are considered interactions (i.e., meet the two requirements of the preceding sentence) are displayed.